archinstall/default_profiles/desktops/hyprland.py

```diff
@@ -24,12 +24,12 @@ def packages(self) -> list[str]:
 			"dunst",
 			"kitty",
 			"uwsm",
-			"dolphin",
-			"wofi",
+			"thunar",
+			"rofi-hyprland",
```

Comment:
Needs to change https://archlinux.org/packages/extra-testing/x86_64/rofi-wayland/